includes/class-wppo-cli-command.php

```
@@ -0,0 +1,373 @@
+				/* translators: %d: Total items removed */
+				Log::add( sprintf( __( 'Database cleanup (all via WP-CLI): %d items removed', 'performance-optimisation' ), $total ) );
+				/* translators: 1: Cleanup type, 2: Total items removed */
+				WP_CLI::success( sprintf( __( 'Database cleanup completed (%1$s): %2$d total items removed.', 'performance-optimisation' ), $type, $total ) );
+				return;
+			}
+
+			$cleaned_count = 0;
+
+			switch ( $type ) {
+				case 'revisions':
+					$cleaned_count = Database_Cleanup::clean_revisions();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Database_Cleanup' in method 'database'. (undefined)

(StaticAccess)